frontend/nuxt.config.ts

```diff
@@ -167,13 +66,39 @@ export default defineNuxtConfig({
     "/meta-search": { redirect: { to: "/about", statusCode: 301 } },
     "/external-sources": { redirect: { to: "/about", statusCode: 301 } },
   },
-  robots,
+  /**
+   * Robots.txt rules are configured here via the \@nuxtjs/robots package.
+   * @see {@link https://nuxtseo.com/robots/guides/nuxt-config|Robots Config Rules}
+   */
+  robots: {
+    disallow: ["/search", "/search/audio", "/search/image"],
+    groups: [
+      ...disallowedBots.map((bot) => ({
+        userAgent: [bot],
+        disallow: ["/"], // block bots from all routes
+      })),
+    ],
+  },
   tailwindcss: {
     cssPath: "~/styles/tailwind.css",
   },
   i18n: {
-    baseUrl: import.meta.env.SITE_URL,
-    locales: openverseLocales,
+    locales: [
+      {
+        /* Nuxt i18n fields */
+
+        code: "en", // unique identifier for the locale in Vue i18n
+        dir: "ltr",
+        file: "en.json",
+        iso: "en", // used for SEO purposes (html lang attribute)
+
+        /* Custom fields */
+
+        name: "English",
```

Comment:
BTW, I noticed this incidentally when looking through the `AppConfig` and `RuntimeConfig` types yesterday, the i18n module actually defines `name` as a property now, so it isn't a custom field anymore :open_mouth:

Nothing to change in this PR, these comments aren't new, but just thought I would mention it.

Reply:
Removed 👍